File: EntertainmentNetworkServer/EntertainmentNetworkServer/src/main/java/isd/dp/ua/EntertainmentNetworkServer/Interfaces/IFloorService.java
package isd.dp.ua.EntertainmentNetworkServer.Interfaces;

import java.math.BigDecimal;
import java.util.List;

import javax.jws.WebService;

import isd.dp.ua.EntertainmentNetworkServer.Models.*;
import isd.dp.ua.EntertainmentNetworkServer.ServiceMessages.*;

@WebService
public interface IFloorService 
{
	void addFloor(AddFloorRequest transientInstance);

	Floor findFloorById(BigDecimal id);
	
	public List<Floor> getFloors();
	
	Floor mergeFloor(MergeFloorRequest detachedInstance);
	
	void removeFloor(BigDecimal id);
}

File: EntertainmentNetworkClient/EntertainmentNetwork.DAL/Models/Interfaces/IFloorService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EntertainmentNetwork.DAL.Models.Interfaces
{
    public interface IFloorService
    {
        Task AddFloor(IFloor seat);

        Task<IFloor> FindFloorById(decimal id);

        Task<List<IFloor>> GetFloors();

        Task<IFloor> MergeFloor(IFloor floor);

        Task RemoveFloor(decimal id);
    }
}
